commands/info.js

```diff
@@ -162,7 +137,7 @@ function locateCustomSheetLocation(drug) {
 
   // Loop through the locationsArray to find the location of a given substance
   for (let i = 0; i < locationsArray.length; i++) {
-    if (locationsArray[i].name == drug) {
+    if (locationsArray[i].name.toLowerCase() == drug) {
```

Comment:
Might as well do `drug.toLowerCase()` just to ensure case sensitivity both directions

Reply:
that's already done in the parseSubstanceName function before it gets passed here, but i can move it just here to be more clear sure